### ISSUE TITLE
Streaming audio over the network to Pulseaudio running in Linux

### ISSUE THREAD

**Issue by BullShark** [pelya/pulseaudio-android#8]
I would like to know is it possible to stream audio from Android to Manjaro Linux over the lan using this project? What if I root this phone? Then? Why or why not?

I couldn't find a build for this project, so I installed XServer Xorg because it said it had this project included with it.

**Comment by Aerol** [comment]
+1 here

**Comment by pelya** [comment]
No, this project will only stream audio from Linux to Android, not the
other way. Microphone input is not supported.

**Comment by BullShark** [comment]
> No, this project will only stream audio from Linux to Android, not the other way. Microphone input is not supported.

Will that ever change in the future? And why not? Pulseaudio handles the microphone input and other types of inputs on other platforms.

**Comment by pelya** [comment]
Mainly because I'm lazy, and I can't quote picture the use case for it.
Running Skype on Linux?

**Comment by ranomier** [comment]
It's cumbersome to always switch between my desktop and android clients for mumble and discord.

It basicly means i have a wireless headset.
And because i have ubiquity wifi gear i can run all around my house.

And i can use any headset/earphones i want.
